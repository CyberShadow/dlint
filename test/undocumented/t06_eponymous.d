// Eponymous templates

template Bad1()
{
	int good11() {};

	struct good12 {}

	struct Bad1
	{
		int bad11;
	}
}

/// documented!
struct good1(T)
{
}

private void good2()() {}


/// documented!
template good1()
{
	void good1()() {}
}
