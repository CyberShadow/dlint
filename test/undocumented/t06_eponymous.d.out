undocumented/t06_eponymous.d(3,1): Undocumented public declaration: TemplateDeclaration `Bad1()`
undocumented/t06_eponymous.d(11,7): Undocumented public declaration: VarDeclaration `bad11`
